drafts/bitcartel-payment-disclosure/draft.rst

```diff
@@ -0,0 +1,345 @@
+| Title: Payment Disclosure
+| ZIP: 00? (DRAFT)
+| Replaces: n/a
+| Superseded-by: n/a
+| Editor: Simon Liu <simon@z.cash>
+
+==================
+Payment Disclosure
+==================
+
+Abstract
+--------
+
+This ZIP describes a method of proving that a payment was sent to a shielded address.  In the typical case, this means enabling a sender to present a proof that they transferred funds to a recipient's shielded address.  The method described will be compatible with the Zcash Protocol [PROTOCOL] deployed at the launch of the Zcash network.
+
+This document follows the common conventions defined for ZIPs in [ZIP-1].
+
+.. [PROTOCOL] Zcash Protocol Specification: https://github.com/zcash/zips/blob/master/protocol/protocol.pdf
+.. [ZIP-1] https://github.com/zcash/zips/blob/master/zips/0001.rst
+
+Copyright Notice
+----------------
+This ZIP is published under the MIT License.
+
+Motivation
+----------
+Payment disclosure is useful in a number of situations, for example:
+
+* A sender may need to prove that their payment was sent and received by a recipient.  For example, a customer paid too much for an item and would like to claim a refund from the vendor.
+* A third party needs to verify that a payment between sender and recipient was executed successfully.  For example, a regulator needs to confirm a transfer of funds between two parties.
+
+When a transaction involves only transparent addresses, the sender and recipient can verify payment by examining the blockchain.  A third party can also perform this verification if they know the transparent addresses of the involved parties.
+
+However, if the transaction involves shielded addresses, the blockchain by itself does not contain enough information from which a record of the payment can be reconstructed and verified.
```

Comment:
Grammar nit: "does not contain enough information to allow a record of the payment to be reconstructed and verified."

Reply:
ok